main/res/values/strings.xml

```diff
@@ -246,8 +246,6 @@
     <string name="warn_log_text_fill">Please input some text for your log.</string>
     <string name="warn_load_images">c:geo failed to load images.</string>
     <string name="warn_invalid_mapfile">The selected map file is not a valid mapsforge map file or uses an unsupported version.\n(c:geo currently supports v3 to v5 / v3 only if you are using \"Old Mapfsforge V3\" setting.)\n\nOffline maps are not available.</string>
-    <string name="warn_nonexistant_mapfile">The selected map file does not exist.\nOffline maps are not available.</string>
-    <string name="warn_rendertheme_missing">Map theme not found.</string>
```

Comment:
I wonder, why these error messages are no longer in use. As I swap my SD from time to time I saw them in the past.